app/javascript/app/components/about/about-partners/about-partners.js

```diff
@@ -41,7 +41,8 @@ const partnershipWith = {
         src: climateActionTrackerImage
       },
       description:
-        'joined the Climate Watch development team and provided support in developing concepts for the scope and functions of Climate Watch.'
+        'joined the Climate Watch development team and provided support in developing concepts for the ' +
```

Comment:
We prefer to use always template literals than concatenation: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
I don't know to what extent should we listen to eslint ;> when you use template literals eslint throws "Strings must use singlequote". I used concatenation as the line was too long. Should I revert those changes?

Reply:
I will revert the changes in these cases. It's better to use string literals in my opinion. You could also disable the es-lint must use singlequote errors. But I'm not sure if it worths it.